Mathlib/MeasureTheory/Measure/MeasureSpace.lean

```diff
@@ -1194,6 +1194,13 @@ theorem map_zero (f : α → β) : (0 : Measure α).map f = 0 := by
 theorem map_of_not_aemeasurable {f : α → β} {μ : Measure α} (hf : ¬AEMeasurable f μ) :
     μ.map f = 0 := by simp [map, hf]
 
+theorem aemeasurable_of_map_ne_zero {μ : Measure α} {μ' : Measure β} {f : α → β}
+  (hf : μ.map f = μ') (nezero : μ' ≠ 0) : AEMeasurable f μ := by
```

Comment:
`μ'` does not seem very useful. You should replace `(hf : μ.map f = μ') (nezero : μ' ≠ 0)` with the simpler `(hf : μ.map f ≠ 0)`.